cardinal/world_persona.go

```diff
@@ -11,6 +11,12 @@ import (
 	"pkg.world.dev/world-engine/cardinal/types"
 )
 
+// implements the validator.SignerAddressProvider interface
+func (w *World) GetSignerAddressForPersonaTag(personaTag string) (addr string, err error) {
```

Comment:
I'd prefer not to add this extra method; it's ambiguous with `w.GetSignerForPersonaTag` and doesn't provide much abstraction.